docs/index.md

```diff
@@ -0,0 +1,125 @@
+# Pay with Card
+
+Accept card payments in your app using the PayPal In-App Payments SDK.
```

Comment:
Are there any card types we _don't_ support today? If so it may be worth calling that out here. Looking at [PPaaS](https://ppaas/api/3719176155270329?tab=apiReference&operation_id=post-_v2_checkout_orders_id_confirm_payment_source) for this endpoint it seems like we should cover most everything so maybe calling out the specific types isn't needed. 🤷‍♂️

Reply:
Yeah i'm not sure. Seeing the card logos does make it feel official though that's a good callout.